tests/atoms/GroundedSchemaLocalUTest.cxxtest

```diff
@@ -0,0 +1,96 @@
+/*
+ * tests/atoms/GroundedSchemaLocalUTest.cxxtest
+ *
+ * Copyright (C) 2018 Alexey Potapov
+ * All Rights Reserved
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License v3 as
+ * published by the Free Software Foundation and including the exceptions
+ * at http://opencog.org/wiki/Licenses
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program; if not, write to:
+ * Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#include <opencog/atomspace/AtomSpace.h>
+#include <opencog/atoms/execution/ExecutionOutputLink.h>
+
+using namespace opencog;
+
+#define N as->add_node
+#define L as->add_link
+
+class GroundedSchemaLocalUTest: public CxxTest::TestSuite
+{
+private:
+    AtomSpace* as;
```

Comment:
It's really just my personally taste here and it shouldn't influence other's dev choice but I think tab indentation (or even better tab for indentation and space of alignment) is really cool! And it seems new files tend to be tab indented (well probably cause both Linas and I prefer tabs over space)... But again, people's choices...